server.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const app = express();
+const port = process.env.PUBLIC_PORT || 3000;
```

Comment:
Consider using environment variables for sensitive data like port numbers. This can help improve the security of your application. [important]